test/.eslintrc.json

```diff
@@ -0,0 +1,14 @@
+{
+  "extends": "springworks/mocha",
+  "globals": {
+    "should": false,
+    "sinon": false
+  },
+  "plugins": [
+    "should-promised"
+  ],
+  "rules": {
+    "springworks/top-describe-path": 2,
```

Comment:
This is already set as default, right? https://github.com/Springworks/eslint-config-springworks/blob/0a83cf2d4ad784a01e290298bfeade1ea72222bb/mocha-config.json#L10